gen/src/main.rs

```diff
@@ -151,14 +157,16 @@ fn main() {
                 writeln!(src_lib_rs, "#[path = \"{}/{}.rs\"]", rust_arch, mod_name).unwrap();
                 writeln!(src_lib_rs, "pub mod {};", mod_name).unwrap();
 
-                run_bindgen(
-                    linux_include.to_str().unwrap(),
-                    header_name.to_str().unwrap(),
-                    &mod_rs,
-                    mod_name,
-                    rust_arch,
-                    linux_version,
-                );
+                if !pre_gen {
+                    run_bindgen(
```

Comment:
Does this mean LoongArch code could potentially get outdated if the bindgen version gets bumped by someone else not aware of this?

Reply:
Yep, that is indeed a risk of pre-generation. It would be great if we could switch to automatic generation as soon as possible.